javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+var $countdownDisplay = document.querySelector('.countdown-display');
+
+setInterval(countdown, 1000);
+
+function countdown() {
+  if ($countdownDisplay.textContent > 1) {
+    $countdownDisplay.textContent--;
+  } else {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(setInterval(countdown, 1000));
```

Comment:
You should be passing the `id` of the setInterval you created earlier on line 3.